index.html

```diff
@@ -36,6 +36,186 @@ <h1>GLAD TO SEE <br>YOU!</h1>
+        </div>
+
+        <article>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        </article>
+
+        <ul class="languages">
+        <li><a href="#">html</a></li>
+        <li><a href="#">css</a></li>
+        <li><a href="#">javaScript</a></li>
+        </ul>
+
+
+        <button class="see">See Project</button>
+
+
+    </li>
+
+
+    <li class="work three">
+        <img src="./snapshot-portfolio3.png" alt="project-image3">
+        
+        <h3>TONIC</h3>
+
+        <div class="sub-title">
+        <h4>Canopy</h4>
+            <i class="fas fa-circle"></i><h4 class="back-end"> Back End Dev</h4>
+            <i class="fas fa-circle"></i><h4 class="date2"> 2015</h4>
+        </div>
+
+        <article>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        </article>
+
+        <ul class="languages">
+        <li><a href="#">html</a></li>
+        <li><a href="#">css</a></li>
+        <li><a href="#">javaScript</a></li>
+        </ul>
+
+
+        <button class="see">See Project</button>
+
+    </li>
+
+    <ul class="work four">
+        <img src="./snapshot-portfolio4.png" alt="project-image4">
+        
+        <h3>MULTI-POST <br>STORIES</h3>
+
+        <div class="sub-title">
+        <h4>Canopy</h4>
+        <i class="fas fa-circle"></i><h4 class="back-end"> Back End Dev</h4>
+        <i class="fas fa-circle"></i><h4 class="date2"> 2015</h4>
+        </div>
+
+        <article>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        </article>
+
+        <ul class="languages">
+        <li><a href="#">html</a></li>
+        <li><a href="#">css</a></li>
+        <li><a href="#">javaScript</a></li>
+        </ul>
+
+        <button class="see">See Project</button>
+        
+    </ul>
+
+    
+        <section class="about">
+        <h2>About <br>Myself</h2>
+        <p class="about1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        
+        <div class="contacts"> 
+        <header><a href="#" class="lets-connect">LET'S CONNECT</a></header>  
+        </div>
+
+        
+        <div class="social-media">
+        <ul class="sm">
+            <li><a href="#" aria-label="twitter"><i class="fab fa-twitter"></i></a></li> 
+            <li><a href="#" aria-label="linkedin"><i class="fab fa-linkedin-in"></i></a></li>
+            <li><a href="#" aria-label="medium"><i class="fab fa-medium-m"></i></a></li>
+            <li><a href="#" aria-label="gtihub"><i class="fab fa-github"></i></a></li> 
+        </ul>
+        </div>
+    
+        <button class="see">Get My Resume</button>
+    </li>
+
```

Comment:
- [ ] Add the necessary styles to the about section. Compare your page with the original
<div class="flex">
<img src="https://user-images.githubusercontent.com/65501535/133996077-c5c94876-6fe0-4556-b009-4a77c2efa908.png"  style="width: 50%;"/>
<img src="https://user-images.githubusercontent.com/65501535/133996275-f3d9f267-8870-423e-b3cc-ca484692b0b1.png"  style="width: 50%;"/>
</div>